ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -262,6 +262,22 @@ objects:
       - -c
       - |
         #!/bin/bash
+        function queue() {
+          local TARGET="${1}"
+          shift
+          local LIVE="$(jobs | wc -l)"
+          while [[ "${LIVE}" -ge 10 ]]; do
+            sleep 1
+            LIVE="$(jobs | wc -l)"
+          done
+          echo "${@}"
```

Comment:
This seemed interesting, but it makes the logs a bit noisier than they used to be.  I'm fine dropping the `echo` if folks want quieter logs for these jobs.